automatic/gimp/tools/chocolateyInstall.ps1

```diff
@@ -1,39 +1,15 @@
 ﻿$ErrorActionPreference = 'Stop';
 
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
 $packageArgs = @{
-  packageName    = 'gimp'
+  packageName    = $env:ChocolateyPackageName
   fileType       = 'exe'
+  file           = "$toolsPath\gimp-2.8.22-setup.exe"
   softwareName   = 'GIMP'
-
-  checksum       = 'a2e52129a28feec1ee3f22f5aaf9bdecbb02d51af6da408ace0a2ac2e0365c8b'
-  checksum64     = 'a2e52129a28feec1ee3f22f5aaf9bdecbb02d51af6da408ace0a2ac2e0365c8b'
-  checksumType   = 'sha256'
-  checksumType64 = 'sha256'
-
-  # The installer contains both 32-bit and 64-bit versions of GIMP, and will automatically use the appropriate one.
-  # Text quoted from <https://www.gimp.org/downloads/>, look below download buttons.
-  url            = 'https://download.gimp.org/mirror/pub/gimp/v2.8/windows/gimp-2.8.22-setup.exe'
-  url64bit       = 'https://download.gimp.org/mirror/pub/gimp/v2.8/windows/gimp-2.8.22-setup.exe'
-
-  silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
+  silentArgs     = "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /LOG=`"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).InnoSetup.log`""
   validExitCodes = @(0)
 }
-$version = '2.8.22'
-$gimpRegistryPath = $(
-  'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\GIMP-2_is1'
-)
 
-if (Test-Path $gimpRegistryPath) {
-  $installedVersion = (
-    Get-ItemProperty -Path $gimpRegistryPath -Name 'DisplayVersion'
-  ).DisplayVersion
-}
-
-if ($installedVersion -eq $version) {
-  Write-Output $(
-    "GIMP $installedVersion is already installed. " +
-    "Skipping download and installation."
-  )
-} else {
-  Install-ChocolateyPackage @packageArgs
-}
+Install-ChocolateyInstallPackage @packageArgs
+Remove-Item -Force -ea 0 "$toolsPath\*.exe","$toolsPath\*.ignore"
```

Comment:
This needs that line with `touch` if rm failed to delete for some reason.

Reply:
I'll update, but can't use `touch` as that isn't available in POSHv2